content/en/platform/corda/5.0/deploying-operating/deploying/bootstrapping.md

```diff
@@ -3,17 +3,22 @@ date: '2023-05-11'
 title: "Manual Bootstrapping"
 menu:
   corda5:
-    parent: corda5-cluster-deploy
+    parent: corda5-cluster-deploying
     identifier: corda5-cluster-bootstrapping
     weight: 2000
 section_menu: corda5
 ---
 
 By default, the Corda installation process automatically performs various setup actions in Kafka and the database, and for Corda RBAC.
-If you require additional control, you can disable these automatic setup processes and an administrator can manually perform the actions
-with the assistance of the [Corda CLI]({{< relref "../tooling/installing-corda-cli.md" >}}).
+If you require additional control, you can disable these automatic setup processes and an administrator can manually perform the actions with the assistance of the [Corda CLI]({{< relref "../tooling/installing-corda-cli.md" >}}).
 
-## Kafka
+This section describes how to configure the following in a YAML file:
+* [Kafka]({{< relref "#kafka" >}})
+* [Database]({{< relref "#database" >}})
+* [RBAC Roles]({{< relref "#rbac-roles" >}})
+
+When you have configured all required values in the YAML file, you can [Deploy Corda]({{< relref "./deploying.md#deployment" >}}).
```

Comment:
Misleading. Most of the configuration is *not* done through a YAML file with manual bootstrapping. It is done by using the CLI (and other tools) to interact directly with the systems to be configured. Where snippets of YAML are included they are largely to disable the automatic bootstrapping. The salt and passphrase are the exception to this. If they've been used as part of manual bootstrapping to encrypt data stored in the configuration database, then we need to make sure those same values are passed to Corda via the configuration YAML to decrypt them.